internal/cli/util/helm/helm.go

```diff
@@ -313,11 +314,46 @@ func (i *InstallOpts) tryUninstall(cfg *action.Configuration) error {
 	}()
 
 	if _, err := client.Run(i.Name); err != nil {
-		return err
+		// Remove secrets left over when uninstalling kubeblocks, when addon CRD is uninstalled before kubeblocks.
+		secretCount, errRemove := i.RemoveRemainSecrets(cfg)
```

Comment:
add a field in `InstallOpts` to set if `forceUninstall`, if true, delete the secret.